src/coresyf_image_crop/tests/test_image_crop.py

```diff
@@ -37,3 +36,19 @@ def test_get_shapefile_crs(self):
         epsg_code = get_shapefile_crs(self.data_source)
         self.assertEqual(epsg_code, self.epsg_code)
 
+    def test_apply_buffer_to_polygon(self):
+        buffer_crs = get_raster_crs(gdal.Open(self.test_image_4236))
```

Comment:
Why to not just define a constant dummy buffer CRS? This way we are adding unneeded dependencies from functions which are tested in another tests. This goes against the reason by which unit tests are _unit_: they allow to test units (functions, classes, modules, etc) functions independently. Here is not the case, if the `get_raster_crs` as some incorrection this test can fail in despite of `apply_buffer_to_polygon_ is correctly implemented.